components/forms/cct/CctCalcCreate.tsx

```diff
@@ -111,352 +110,372 @@ export function CctCalcCreate() {
-                                              value.endsWith("%")
-                                                ? Number(value.slice(0, -1))
-                                                : value;
-                                            if (parsedValue) {
-                                              setFieldValue(
-                                                field,
-                                                parsedValue / 100
-                                              );
-                                            } else {
-                                              setFieldValue(field, null);
-                                            }
-                                          }}
-                                          error={
-                                            errors.changes &&
-                                            (
-                                              errors.changes as CctCalculationErrors["changes"]
-                                            )?.[index]?.wte
-                                          }
-                                          options={fteOptions}
-                                          name={`changes[${index}].wte`}
-                                          label="Proposed WTE (%)"
-                                          isMulti={false}
-                                          closeMenuOnSelect={true}
-                                          isCreatable={true}
-                                          defaultOption={
-                                            values.changes[index].wte
-                                              ? {
-                                                  value:
-                                                    values.changes[index].wte,
-                                                  label: `${
-                                                    (values.changes[index]
-                                                      .wte as number) * 100
-                                                  }%`
+                                          {!(
+                                            errors.changes as CctCalculationErrors["changes"]
+                                          )?.[index]?.startDate &&
+                                            dayjs(
+                                              values.changes[index].startDate
+                                            ) <
+                                              dayjs()
+                                                .add(16, "week")
+                                                .subtract(1, "day") && (
+                                              <span data-cy="start-short-notice-warn">
+                                                <FieldWarningMsg warningMsg="Actioning a change in WTE hours with less than 16 weeks notice might not be possible logistically (e.g. rota constraints)." />
```

Comment:
Maybe simplify this to something like:

It may not be possible to change WTE hours with less than 16 week's notice (due to e.g. rota constraints).